roles/ood_firewall_and_services/tasks/main.yaml

```diff
@@ -5,6 +5,16 @@
     name: httpd24-httpd
     enabled: yes
 
+- name: install the latest version of Python
+  yum:
+    name: python
+    state: latest
+
+- name: install the latest version of firewalld
+  yum:
+    name: firewalld
+    state: latest
+
```

Comment:
What motivates the the firewalld install? It's install appears to prevent srun from submitting jobs, at least with the default config.  Is this something we need on ood at present?